src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,168 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final int SIZE_DIVIDER = 2;
+    private static final int SIZE_SHIFTER = 1;
+    private static final int ZERO_SIZE = 0;
```

Comment:
concept of zero is unlikely to change :)) You can hardcode such self-explaining values